### ISSUE TITLE
What computer do you use/recommend for mobile app development?

### ISSUE THREAD

**Issue by nickpeihl** [incanus/ama#8]

**Comment by incanus** [comment]
Neat question @npeihl. 

Since 2002, I've been a laptop-only Mac user, starting with a Titanium PowerBook G4 and continuing with Apple's higher-end machines since then (PowerBook G4, MacBook Pro). 

My last single machine was a 2009 MacBook Pro 17", fully maxed out with dual-core 2.66GHz Core 2 Duo, 1.5TB of disk (dual SSD primary / HDD secondary in optical bay), and 8GB of RAM. This served me for five years. 

In 2014, due to a lot of C++ work, I needed a faster compile time, so for work I got a MacBook Pro 15" with quad-core i7 2.6GHz, SSD, and 16GB RAM. This machine is still serving me well. When at home (~50% of the time) I use an older ~2009 24" Cinema Display for a bit more screen space alongside Apple's Bluetooth keyboard and Magic Trackpad. The whole setup is so easy to switch between mobile and desk work. 

I needed more space for photos and other projects (hence the 1.5TB in the old machine), so I also got a personal MacBook Air 11", which I love, but it's not quite fast enough or large enough for everyday work. It's so portable though and I love it dearly (using it to write this now). 

Dropbox & GitHub for projects and iCloud for browser tabs help me keep things in sync between the machines.
